### PR TITLE
docs(Breadcrumbs): add readme

### PR DESCRIPTION
**Pull request by Feverqwe**

### CODE REVIEW

File: src/components/Breadcrumbs/README.md [hunk 1]
```diff
@@ -1,43 +1,345 @@
...
-| renderItemContent        | `Function` |          |         | Custom render function of N+1 item `(item: BreadcrumbsItem, isCurrent: boolean, isPrevCurrent: boolean) => React.ReactNode;` |
-| renderItemDivider        | `Function` |          |         | Custom render function of items separator `() => React.ReactNode;`                                                           |
-| lastDisplayedItemsCount  | `Enum`     |    ✓     |         | Number of items to display after items collapse control: `LastDisplayedItemsCount`                                           |
-| firstDisplayedItemsCount | `Enum`     |    ✓     |         | Number of items to display before items collapse control: `FirstDisplayedItemsCount`                                         |
-| popupStyle               | `String`   |          |         | Style of collapsed items popup `staircase`                                                                                   |
+```tsx
+import {Breadcrumbs} from '@gravity-ui/uikit';
+```
+
+`Breadcrumbs` can collapse breadcrumbs that cause overflow.
```
Comment (korvin89) [inline src/components/Breadcrumbs/README.md:11]: I think we need a little bit more suitable description. [Smth like this](https://mui.com/material-ui/react-breadcrumbs/)

File: src/components/Breadcrumbs/README.md [hunk 2]
```diff
@@ -1,43 +1,345 @@
...
+
+## Appearance
```
Comment (korvin89) [inline src/components/Breadcrumbs/README.md:13]: U can write about lastDisplayedItemsCount and firstDisplayedItemsCount either

File: src/components/Breadcrumbs/README.md [hunk 3]
```diff
@@ -1,43 +1,345 @@
...
+    renderItemDivider={() => '>'}
+    firstDisplayedItemsCount={FirstDisplayedItemsCount.One}
+    lastDisplayedItemsCount={LastDisplayedItemsCount.One}
   />
 );
 ```
+
+<!--/GITHUB_BLOCK-->
+
+### Custom title
```
Comment (korvin89) [inline src/components/Breadcrumbs/README.md:218]: Seems like a part of `BreadcrumbsItem` item menu, but not the part of the Appearance

File: src/components/Breadcrumbs/README.md (hunk 2 above)
Comment (Feverqwe) [reply src/components/Breadcrumbs/README.md:13]: I think it extra info, I update description, if it not enough I can add info about it.

**Review by korvin89 — APPROVED** [review]

**Review by vrozaev — APPROVED** [review]

**Pull request merged**